src/retrieve-validation/node.ts

```diff
@@ -15,11 +15,19 @@ const retrieveInNode = async (url: string, timeout: number): Promise<W3CCSSValid
 				let data = '';
 
 				res.on('data', (chunk) => {
-					data += chunk;
+					try {
+						data += chunk;
+					} catch (e) {
+						reject(e);
+					}
```

Comment:
Can this actually ever throw an error? If so, under what circumstances?

Reply:
I seriously doubt it, the only thing I imagined was some error resulting from attempted coercion if chunk was some unexpected data type. I wrapped it in the case that it could ever throw an error, it would be rejected.

Reply:
Instead of try/catch, can we do `String(chunk)` instead?

Reply:
We could, but it might be worth getting a report about an error/rejected promise if it did actually cause an error.

Reply:
Okay, that's fair; then I would just rename `e` to `error` for clarity.